### ISSUE TITLE
Example is broken

### ISSUE THREAD

**Issue by lucianlature** [typeorm/javascript-example#1]
When I install the latest dependencies for the examples, and then try to run `node src/app3-es6/index.js`, I get:

```
name: 'RepositoryNotFoundError',
message: 'No repository for "Post" was found. Looks like this entity is not registered in current "default" connection?' }
```

**Comment by shodhan** [comment]
yeah i get the same error. has anyone find whats wrong!

**Comment by chriszrc** [comment]
I encountered this error when typeorm can't find the entity class. Usually a misconfigured

```
 entities: [
        __dirname + "/entity/*.js"
    ],
```

Could the path be wrong with this example?

**Comment by pleerock** [comment]
I'll update all dependencies and fix this issue once 0.1.0 is released

**Comment by pleerock** [comment]
Looks like only ES6 is broken, Ive fixed it.
